plugins/modules/system/parted.py

```diff
@@ -610,8 +618,9 @@ def main():
 
         # Create partition if required
         if part_type and not part_exists(current_parts, 'num', number):
-            script += "mkpart %s %s %s " % (
+            script += "mkpart %s %s%s %s " % (
                 part_type,
+                f"{fs_type} " if fs_type is not None else '',
```

Comment:
```suggestion
                '%s ' % fs_type if fs_type is not None else '',
```

Reply:
For clarity I'd add parantheses:
```suggestion
                ('{fs_type} ' % fs_type) if fs_type is not None else '',
```
The `f''` syntax is Python 3.6+ only (Ansible also supports 2.7 and 3.5).